### PR TITLE
The X icon doesn't remain when the text is too long

### LINKED ISSUE #83: The X icon doesn't remain when the text is too long
## Expected Behavior
The X icon in selected items will remain even if the text is too long

## Actual Behavior
The X icon doesn't remain when the text is too long

## Steps to Reproduce the Problem

  1. put long string in the item label
  2. move it to selected list
  3. check if X icon exist

### PR DESCRIPTION
**Pull request by michalsabag**
Fixes #83 
The X icon in selected items is remain even if the text is too long with tooltip.

### CODE REVIEW

**Comment by liorheber** [comment]
Thanks, some work needs to be done:

1. build is broken.
2. new tests were not added for the new feature.
3. please minimize the code duplication.
3. please add a screenshot of the solution to the PR.

**Comment by michalsabag** [comment]
image

**Comment by liorheber** [comment]
Screenshot doesn't look too great.
The lines are being cut, anything we can do about that?
Additionally, be sure that the padding in the options to the left is aligned on both sides.
It looks like the text goes all the way to the right without padding.

File: tests/components/items/item_label.spec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  test("default snapshot", () => {
+    const renderer = new ShallowRenderer();
+    const tree = renderer.render(<ItemLabel />);
+    expect(tree).toMatchSnapshot();
+  });
+
+  test("snapshot with label text", () => {
+    const label = "hi";
+    const renderer = new ShallowRenderer();
+    const tree = renderer.render(<ItemLabel label={label} />);
```
Comment (liorheber) [inline tests/components/items/item_label.spec.js]: can we have a snapshot with overflow and without?
Seems like the title is there 100% of the time based on the current snapshot

**Pull request merged**